db-migration/src/main/resources/db/migration/V3__relational_table_customers_cars.sql

```diff
@@ -0,0 +1,11 @@
+create table customers_cars
```

Comment:
CREATE TABLE IF NOT EXIST. Look at the previously created scripts

Reply:
Ok